packages/react-tracking-pixel/src/TrackingPixel.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import isEqual from 'lodash/isEqual';
```

Comment:
Add any external libs to `package.json#dependency`.  Otherwise consumer builds will fail with confusing errors.